### PR TITLE
Placeholder text, redux

### PR DESCRIPTION
**Pull request by andrewhao**
[Gyazo](http://gyazo.com/c7a9c64feb36e85a5608dfcf304eacef)

@mwynholds showed me code that he wrote doing this exact thing. We should use that.

### CODE REVIEW

File: lib/views/index.jade [hunk 1]
```diff
@@ -6,7 +6,7 @@ include includes/header
     .span12.text-center
       .mega-text
         form.form-inline(id='create-board', method='post', action='/boards')
-          input.mega-text__input(type='text', name='name', value="#{userIdentity.displayName}'s board" placeholder="#{userIdentity.displayName}'s board", required=true)
+          input#board-name.mega-text__input(type='text', name='name', value="#{userIdentity.displayName}'s board" value="#{userIdentity.displayName}'s board", required=true)
```
Comment (crsven) [inline lib/views/index.jade]: double `value`

File: assets/js/index.coffee [hunk 2]
```diff
@@ -1,4 +1,5 @@
 $ ->
+  # Set up placeholder widget on the form creation field.
```
Comment (crsven) [inline assets/js/index.coffee]: instead of comment, can we just rename it to `addPlaceholder` or something?

File: lib/views/index.jade (hunk 1 above)
Comment (andrewhao) [inline lib/views/index.jade]: Oops! Thanks. Fixed.

File: assets/js/index.coffee (hunk 2 above)
Comment (andrewhao) [inline assets/js/index.coffee]: Done.

**Comment by crsven** [comment]
LGTM

**Pull request merged**